deploy/lib/ml_rest.rb

```diff
@@ -17,7 +17,6 @@ def initialize(options)
       @request = {}
       @gmt_offset = Time.now.gmt_offset
       @server_version = options[:server_version]
-      @rest_protocol = "http#{options[:use_https_for_rest] ? 's' : ''}"
```

Comment:
This looks like it should not have been removed. Perhaps the code base you worked from was a bit out of date? Try updating and rebasing.